manage_breast_screening/templates/wizard_step.jinja

```diff
@@ -90,8 +90,12 @@
         </div>
     </div>
   </form>
+  {% endif %}
 
+  {% if cannot_continue_link %}
+    <p class="nhsuk-u-margin-top-x9"><a class="nhsuk-link nhsuk-link--no-visited-state" href="{{ cannot_continue_link.href }}">{{ cannot_continue_link.text }}</a></p>
```

Comment:
`nhsuk-u-margin-top-x9` - typo?

Aside: this 'link' is used repeatedly in the app - I wonder if we should abstract as reusable thing?

Reply:
Have removed this now, as it's unnecessary